examples/apps/data-object-grid/src/dataObjectRegistry.ts

```diff
@@ -30,10 +30,16 @@ interface ISingleHandleItem {
 function createSingleHandleItem(subFactory: IFluidDataStoreFactory) {
 	return async (context: IFluidDataStoreContext): Promise<ISingleHandleItem> => {
 		const packagePath = [...context.packagePath, subFactory.type];
-		const router = await context.containerRuntime.createDataStore(packagePath);
-		const object = await requestFluidObject<IFluidLoadable>(router, "/");
+		const dataStore = await context.containerRuntime.createDataStore(packagePath);
+
+		if (dataStore.entryPoint !== undefined) {
```

Comment:
When can we remove this condition/fallback?

Reply:
Do you need it even now?  As long as all the current example data objects have entrypoints seems fine to remove to me - we don't need to demonstrate backcompat support with this example.